.changelog/11774.txt

```diff
@@ -0,0 +1,12 @@
+```release-note:enhancement
+resource/aws_ssm_maintenance_window_task: Add `cloudwatch_config` argument
+```
+
+```release-note:enhancement
+resource/aws_ssm_maintenance_window_task: Add `document_version` argument
+```
+
+```release-note:enhancement
+resource/aws_ssm_maintenance_window_task: Add plan time validation to `task_type`, `service_role_arn`, `targets.notification_arn`, `targets.service_role_arn`, `max_concurrency`, `max_errors`, `targets`, `priority`, 
+`task_invocation_parameters.run_command_parameters.comment`, `task_invocation_parameters.run_command_parameters.document_hash`, `task_invocation_parameters.run_command_parameters.timeout_seconds`, `task_invocation_parameters.run_command_parameters.notification_config.notification_events` 
```

Comment:
Newlines should not be added in CHANGELOG entries. Reference: https://github.com/hashicorp/terraform-provider-aws/issues/17135

```suggestion
resource/aws_ssm_maintenance_window_task: Add plan time validation to `max_concurrency`, `max_errors`, `priority`, `service_role_arn`, `targets`, `targets.notification_arn`, `targets.service_role_arn`, `task_type`, `task_invocation_parameters.run_command_parameters.comment`, `task_invocation_parameters.run_command_parameters.document_hash`, `task_invocation_parameters.run_command_parameters.timeout_seconds`, and `task_invocation_parameters.run_command_parameters.notification_config.notification_events` arguments
```